project/FMPP.scala

```diff
@@ -26,8 +26,8 @@ object FMPP {
   val FmppConfig = config("fmpp").hide
   def fmppTask = Def.task {
     val s = streams.value
-    val output = sourceManaged.value
-    val fmppSrc = sourceDirectory.value / "scala"
+    val output = (Compile / sourceManaged).value
```

Comment:
Can you explain why it was necessary to add `Compile /` in a bunch of places?

Reply:
its related to this issue  https://github.com/scoverage/sbt-scoverage/issues/105 wrapping the output in compile seems to solve the issue